Simplenote/NoticePresenter.swift

```diff
@@ -105,13 +108,17 @@ class NoticePresenter {
 
     // MARK: Dismissing Methods
     //
-    func dismissNotification(completion: @escaping () -> Void) {
+    func dismissNotification(withDuration duration: TimeInterval?, completion: @escaping () -> Void) {
```

Comment:
I **think**  (note the think, leaving room for error !! 😂) proper Swift would be:

```
    func dismissNotification(duration: TimeInterval?, completion: @escaping () -> Void) {
Write
```

[Ref. here!!](https://swift.org/documentation/api-design-guidelines/) (**"Argument Labels"**)

Reply:
hmmm..... that is a tricky one conventionally.

I chose to put withDuration because of this part of the guideline
>The argument label should normally begin at the preposition, e.g. x.removeBoxes(havingLength: 12).

But I could see that it could also be
`dismissNotificationWith(duration: TimeInterval?, onCompletion:.....)`
based on the exception
>An exception arises when the first two arguments represent parts of a single abstraction.
a.move(toX: b, y: c)
a.fade(fromRed: b, green: c, blue: d)

>In such cases, begin the argument label after the preposition, to keep the abstraction clear.
a.moveTo(x: b, y: c)
a.fadeFrom(red: b, green: c, blue: d)

I've changed it to your suggestion.  Will have to think about this one 🤔

Reply:
Hmmm.... more complex.  I went to make this change and noticed something:
func dismissNotification(**withDuration** duration: TimeInterval, completion: @escaping () -> Void) {

calls 
UIView.animate(**withDuration:** duration, delay: .zero, options: [.allowUserInteraction]) {

Should we follow the guidelines that apple is using in their API?